pkg/runtime/runtime.go

```diff
@@ -222,6 +239,24 @@ func (r *Runtime) Upgrade(fs apfs.FS, newRt *Runtime, config *CommonConfig) ([]A
 	return newComponents, nil
 }
 
+func (r *RuntimeSpec) install(ctx context.Context, f apkube.Factory, cloneOpts *apgit.CloneOptions, runtimeName string, valuesProvider HelmValuesProvider) error {
+	for _, component := range r.Components {
```

Comment:
the components here are already filtered - so no need to check the installFeatures again